### PR TITLE
Added support for decimal seconds

### PR DESCRIPTION
**Pull request by siaavenza**

### CODE REVIEW

File: geolucidate/tests/tests.py [hunk 1]
```diff
@@ -87,7 +91,7 @@ def test_parser():
         (u"70 ° 57N 070 ° 05W", ['N', '70', '57', '00', 'W', '070', '05', '00']),
         (u"""(45º10'17"N 076º23'46"W)""", ['N', '45', '10', '17', 'W', '076', '23', '46']),
         # Note that the degree and minute punctuation are actually backwards; we support it anyway.
-        (u"""(45º10"17'N 076º23"46'W" """, ['N', '45', '10', '17', 'W', '076', '23', '46']),
+        # (u"""(45º10"17'N 076º23"46'W")""", ['N', '45', '10', '17', 'W', '076', '23', '46']),
```
Comment (siaavenza) [inline geolucidate/tests/tests.py:124]: @AvenzaOleg this one was missed when you created the new tests. I'm thinking we don't support this format?

the minute/second marks are switched

File: geolucidate/tests/tests.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
     ('491500N 1230720W', '49.250000,-123.122222'),
     ('5046.6N / 06829.2W', '50.777,-68.487'),
     ('5734.8 N / 10006.2 W', '57.580,-100.103'),
-    ('4952.013N / 09548.474W', '49.867,-95.808'),
+    ('(4952.013N / 09548.474W)', '49.867,-95.808'),
```
Comment (siaavenza) [inline geolucidate/tests/tests.py:44]: This line originally had brackets. I added them back

File: geolucidate/parser.py [hunk 3]
```diff
@@ -29,9 +29,11 @@
     $
   """, re.VERBOSE | re.UNICODE | re.IGNORECASE)
 
-degree_min_sec_re = re.compile(r"""\b
+degree_min_sec_re = re.compile(r"""
+    ^
+    \(?
     # Latitude direction, first position: one of N, S, NORTH, SOUTH
-    ((?P<latdir>NORTH|SOUTH|[NS])\ ?)?
+    (((?P<latdir>NORTH|SOUTH|[NS])\ ?)|(?P<latsign>-))?
```
Comment (siaavenza) [inline geolucidate/parser.py:36]: Added support for `-` delimiter. They're called `latsign` and `longsign`. Used in  [_cleanup()](https://github.com/Avenza/geolucidate/pull/5/files#diff-ec126768a1469c3f7965134da04b11fcR49)

File: geolucidate/tests/tests.py (hunk 1 above)
Comment (AvenzaOleg) [reply geolucidate/tests/tests.py:124]: It does not look like a valid format, let's keep it commented

**Pull request merged**